processor/tailsamplingprocessor/README.md

```diff
@@ -163,4 +167,7 @@ The probabilistic sampling policy makes decision based upon the trace ID, so wai
 You are already incurring the cost of running the tail sampling processor, adding the probabilistic policy will be negligible.
 Additionally, using the policy within the tail sampling processor will ensure traces that are sampled by other policies will not be dropped.
 
-[probabilistic_sampling_processor]: ../probabilisticsamplerprocessor
+[beta]: https://github.com/open-telemetry/opentelemetry-collector-contrib#beta
+[core]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
```

Comment:
Can we link from the table?

Reply:
These are linked in the table, i just defined the aliases for the links at the bottom